jobs/rosmap-pcc-paired/job.json

```diff
@@ -0,0 +1,714 @@
+{
+    "cwl_wf_url": "https://github.com/Sage-Bionetworks-Workflows/dockstore-workflow-rnaseq",
```

Comment:
Will update this URL to point to tagged release once we have figured out that issue